src/service/resources-impl.js

```diff
@@ -1329,7 +1329,7 @@ export class Resources {
       }
       // Nothing to do.
       if (opt_callback) {
-        opt_callback(/* hasSizeChanged */false);
+        opt_callback(/* success */ true);
```

Comment:
Didn't find it documented as such in any jsdocs - it's a callback with an anonymous param. However, since it's used mainly for the promise resolve/reject, I find it hard to explain that the situation we have ("size is already satisfied") should be a rejection.